examples/azure-tf-example/porter.yaml

```diff
@@ -0,0 +1,150 @@
+      input: false
+      backendConfig:
+        key: "{{ bundle.name }}.tfstate"
+        storage_account_name: "{{ bundle.parameters.storage_account_name }}"
+        container_name: "{{ bundle.parameters.storage_container_name }}"
+        access_key: "{{ bundle.outputs.STORAGE_ACCOUNT_KEY }}"
+      vars:
+        subscription_id: "{{bundle.credentials.subscription_id}}"
+        tenant_id: "{{bundle.credentials.tenant_id}}"
+        client_id: "{{bundle.credentials.client_id}}"
+        client_secret: "{{bundle.credentials.client_secret}}" 
+        database_name: "{{bundle.parameters.database-name}}"
+        resource_group_name: "{{bundle.parameters.resource_group_name}}"
+        resource_group_location: "{{bundle.parameters.location}}"
+      outputs:
+      - name: cosmos-db-uri
+      - name: eventhubs_connection_string
+    
+upgrade:
+  - terraform:
+      description: "Update Azure CosmosDB and Event Hubs"
+      autoApprove: true
+      input: false
+      vars:
+        subscription_id: "{{bundle.credentials.subscription_id}}"
+        tenant_id: "{{bundle.credentials.tenant_id}}"
+        client_id: "{{bundle.credentials.client_id}}"
+        client_secret: "{{bundle.credentials.client_secret}}" 
+        database_name: "{{bundle.parameters.database-name}}"
+        resource_group_name: "{{bundle.parameters.resource_group_name}}"
+        resource_group_location: "{{bundle.parameters.location}}"
+      backendConfig:
+        key: "{{ bundle.name }}.tfstate"
+        storage_account_name: "{{ bundle.parameters.storage_account_name }}"
+        container_name: "{{ bundle.parameters.storage_container_name }}"
+        access_key: "{{ bundle.parameters.tf_storage_account_key }}"
+
+uninstall:
+  - terraform:
+      description: "Remove Azure CosmosDB and Event Hubs"
+      autoApprove: true
+      backendConfig:
+        key: "{{ bundle.name }}.tfstate"
+        storage_account_name: "{{ bundle.parameters.storage_account_name }}"
+        container_name: "{{ bundle.parameters.storage_container_name }}"
+        access_key: "{{ bundle.parameters.tf_storage_account_key }}"
+      vars:
+        subscription_id: "{{bundle.credentials.subscription_id}}"
+        tenant_id: "{{bundle.credentials.tenant_id}}"
+        client_id: "{{bundle.credentials.client_id}}"
+        client_secret: "{{bundle.credentials.client_secret}}" 
+        database_name: "{{bundle.parameters.database-name}}"
+        resource_group_name: "{{bundle.parameters.resource_group_name}}"
+        resource_group_location: "{{bundle.parameters.location}}"
+  - azure:
+      description: "Remove the Azure Storage Account"
+      name: porter-tf-storage
+      resourceGroup: "{{ bundle.parameters.storage_rg }}"
+      type: storage
+      parameters:
+        location: "{{ bundle.parameters.location }}"
+        storageAccountName: "{{ bundle.parameters.storage_account_name }}"
+        storageContainerName: "{{ bundle.parameters.storage_container_name }}"
```

Comment:
We could add a bundle outputs section here:
```
outputs:
- name: cosmos-db-uri
  type: string
- name: eventhubs_connection_string
  type: string
```

If this is done, we could update docs at some point to inform the reader that they can grab a specific output via `porter bundle output show cosmos-db-uri`, etc.  (Depends on https://github.com/deislabs/porter/pull/451)